packages/editor/styles/styles.css

```diff
@@ -903,6 +878,25 @@ del.diffdel {
 .ProseMirror h6 {
   display: flex;
   align-items: center;
+  justify-content: flex-start;
+}
+
+.ProseMirror h1[style*="text-align: center"],
+.ProseMirror h2[style*="text-align: center"],
+.ProseMirror h3[style*="text-align: center"],
+.ProseMirror h4[style*="text-align: center"],
+.ProseMirror h5[style*="text-align: center"],
+.ProseMirror h6[style*="text-align: center"] {
```

Comment:
Why not use `class` instead?

Reply:
the `setTextAlign` editor command sets the inline style `text-align`, that is what I'm using for the CSS selector

Not sure I understand how th use `class` instead?